test/tests/concrete_ASR_swelling/tests

```diff
@@ -14,7 +14,19 @@
       custom_cmp = 'asr_confined.cmp'
       detail = 'with anisotropic expansion strains, '
     []
-
+    [test_strip_kelvin]
+      type = 'Exodiff'
+      input = 'asr_confined.i'
+      exodiff = 'asr_confined_strip_kelvin_out.e'
+      cli_args = 'AuxVariables/T/initial_condition=308.15 '
+                 'Materials/ASR_expansion/temperature_unit=Kelvin '
+                 'Materials/ASR_expansion/reference_temperature=308.15 '
+                 'Materials/thermal_strain_concrete/stress_free_temperature=308.15 '
+                 'Materials/thermal_strain_steel/stress_free_temperature=308.15 '
+                 'Outputs/file_base=asr_confined_strip_kelvin_out'
+      custom_cmp = 'asr_confined_kelvin.cmp'
```

Comment:
I suspect that the `asr_confied_kelvin.cmp` file you added is nearly identical to the `asr_confined.cmp` file. If so, please delete that file, and just point to the existing `asr_confined.cmp` file. If not, we probably have other issues to look into!

Reply:
Based on our conversation, let's keep this as-is, since this accounts for the difference in temperature fields.